actor-macros/src/lib.rs

```diff
@@ -0,0 +1,137 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, Attribute, DeriveInput, GenericParam, Lit};
+
+#[proc_macro_derive(Actor, attributes(actor))]
+pub fn actor(item: TokenStream) -> TokenStream {
```

Comment:
It might be nice to add some comments on this explaining the default names/attributes/example usage, but the example does a good job so I'm not too worried about it now.

Reply:
Good call